rclcpp/resource/logging.hpp.em

```diff
@@ -87,19 +86,25 @@ def is_supported_feature_combination(feature_combination):
 @[ end for]@
  * \param logger The `rclcpp::Logger` to use
 @[ for param_name, doc_line in feature_combinations[feature_combination].params.items()]@
+@[ if not param_name == 'time_source']@
  * \param @(param_name) @(doc_line)
+@[ else]@
+ * \param clock rclcpp::Clock that will be used to get the time point.
+@[ end if]@
```

Comment:
@BMarchi qq, wouldn't updating the feature combination dict be more robust? This will pick up any `param_name` called `time_source`.

Reply:
Bear in mind that if I modify the dictionaries, I have to recreate all the other methods inside this configure file or modify the global variable, are you okay with that?